src/main/java/seedu/address/MainApp.java

```diff
@@ -94,7 +95,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialData = new AddressBook();
         }
 
-        return new ModelManager(initialData, userPrefs);
+        return new ModelManager(initialData, userPrefs, new ModelHistory());
```

Comment:
possible improvements:
put the initialization of model history in initModelManager or init to be consistent with the original code base